group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -1830,6 +1832,64 @@ public void onLoaded() {
         });
     }
 
+    // TODO: this part is done asynchronously in the old coordinator via a KafkaScheduler#schedule
+    /**
```

Comment:
would like to point out that the existing coordinator does this asynchronously

Reply:
That's a good point. We need to observe how it works. If it causes performance issues, we could move this to the loader and use the loader to load and unload.